deploy/gk-deploy

```diff
@@ -151,13 +151,15 @@ abort() {
     eval_output "${CLI} delete template deploy-heketi 2>&1"
     eval_output "${CLI} delete template heketi 2>&1"
   else
+    eval_output "${CLI} delete clusterrolebinding heketi-sa-view 2>&1"
```

Comment:
What does this line do? Asking because I don't understand. Nothing wrong with the patch.